apps/web/playwright/filters.e2e.ts

```diff
@@ -0,0 +1,696 @@
+
+    const skillOptionId1 = uuidv4();
+    const skillOptionId2 = uuidv4();
+    const skillOptionId3 = uuidv4();
+
+    const form = await routingForms.create({
+      name: formName,
+      userId: owner.id,
+      teamId: membership.teamId,
+      fields: [
+        {
+          type: "select",
+          label: "Location",
+          identifier: "location",
+          required: true,
+          options: [
+            {
+              id: locationOptionId1,
+              label: "New York",
+            },
+            {
+              id: locationOptionId2,
+              label: "London",
+            },
+          ],
+        },
+        {
+          type: "multiselect",
+          label: "Skills",
+          identifier: "skills",
+          required: true,
+          options: [
+            {
+              id: skillOptionId1,
+              label: "JavaScript",
+            },
+            {
+              id: skillOptionId2,
+              label: "TypeScript",
+            },
+            {
+              id: skillOptionId3,
+              label: "React",
+            },
+          ],
+        },
+      ],
+    });
+
+    await prisma.app_RoutingForms_FormResponse.create({
+      data: {
+        formFillerId: "test-filler-1",
+        formId: form.id,
+        response: {
+          [singleSelectFieldId]: {
+            label: "Location",
+            value: locationOptionId1,
+          },
+          [multiSelectFieldId]: {
+            label: "Skills",
+            value: [skillOptionId1, skillOptionId2],
+          },
+        },
+      },
+    });
+
+    await prisma.app_RoutingForms_FormResponse.create({
+      data: {
+        formFillerId: "test-filler-2",
+        formId: form.id,
+        response: {
+          [singleSelectFieldId]: {
+            label: "Location",
+            value: locationOptionId2,
+          },
+          [multiSelectFieldId]: {
+            label: "Skills",
+            value: [skillOptionId2, skillOptionId3],
+          },
+        },
+      },
+    });
+
+    await page.goto(`/insights/routing`);
+
+    await selectFilter(page, "location");
+    await page.getByTestId("filter-popover-trigger-location").click();
+    await page
+      .getByTestId("select-filter-options-location")
+      .getByRole("option", { name: "New York" })
+      .click();
+    await page.keyboard.press("Escape");
+
+    await expect(page.getByText("New York")).toBeVisible();
+    await expect(page.getByText("London")).toBeHidden();
+
+    await clearFilters(page);
+    await selectFilter(page, "skills");
+    await page.getByTestId("filter-popover-trigger-skills").click();
+    await page.getByTestId("select-filter-options-skills").getByRole("option", { name: "React" }).click();
+    await page.keyboard.press("Escape");
+
+    await expect(page.getByText("React")).toBeVisible();
+  });
+
+  test("createdAt filter: should filter by date range", async ({ page, users, routingForms, prisma }) => {
```

Comment:
There's a gotcha on this page. The createdAt filter button is slightly on a different part of the page. On the top right side, there's a button which should probably called "Last 7 days". It's not a part of ActiveFilters component. Check its data-testid and click it properly.